edgelet/edgelet-config/src/lib.rs

```diff
@@ -454,6 +454,21 @@ where
         &self.watchdog
     }
 
+    pub fn compute_settings_digest(&self) -> Result<String, LoadSettingsError> {
```

Comment:
This can probably be a private function?

Reply:
Refactored the code and now this is private.